umh-core/examples/example-config-benthos-monitor.yaml

```
@@ -0,0 +1,57 @@
+              curl -sSL --max-time 1 http://localhost:9000/ping | gzip -c | xxd -p
+              echo "PINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGENDPINGEND"
+              curl -sSL --max-time 1 http://localhost:9000/ready | gzip -c | xxd -p
+              echo "CONFIGENDCONFIGENDCONFIGENDCONFIGENDCONFIGENDCONFIGENDCONFIGENDCONFIGENDCONFIGENDCONFIGENDCONFIGEND"
+              curl -sSL --max-time 1 http://localhost:9000/version | gzip -c | xxd -p
+              echo "VERSIONENDVERSIONENDVERSIONENDVERSIONENDVERSIONENDVERSIONENDVERSIONENDVERSIONENDVERSIONENDVERSIONENDVERSIONEND"
+              curl -sSL --max-time 1 http://localhost:9000/metrics | gzip -c | xxd -p
+              echo "METRICSENDMETRICSENDMETRICSENDMETRICSENDMETRICSENDMETRICSENDMETRICSENDMETRICSENDMETRICSENDMETRICSEND"
+              date +%s%9N
+              echo "ENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDENDEND"
+              sleep 1
+            done
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)